src/ontology/imports/foodon_product_import.owl

```diff
@@ -33599,6 +33605,109 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <obo:IAO_0000115 xml:lang="en">A part of an animal that is leftover after meat has been harvested.</obo:IAO_0000115>
+        <obo:IAO_0000117 xml:lang="en">https://orcid.org/0000-0001-5275-8866</obo:IAO_0000117>
+        <obo:IAO_0000119 xml:lang="en">https://blog.publicgoods.com/what-are-animal-products-and-byproducts/</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T21:15:24Z</dc:date>
+        <rdfs:label xml:lang="en">animal by-product</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004166 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004166">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002574"/>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T21:24:34Z</dc:date>
+        <rdfs:label xml:lang="en">cut of beef, mexican</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004167 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004167">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004166"/>
+        <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000428"/>
+        <obo:IAO_0000115 xml:lang="en">A cut of beef and part of the sub-prime cut known as the chuck.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119>https://rollybrook.com/lexicon-groups.htm</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.mexconnect.com/articles/2398-choice-cut-or-mystery-meat-a-guide-to-mexican-butcher-shops-part-i-beef/</obo:IAO_0000119>
+        <rdfs:label xml:lang="en">diezmillo</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004168 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004168">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03317170"/>
+        <obo:IAO_0000115 xml:lang="en">The process of slow cooking meat to make it easier to chew or cut.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Meat_tenderness</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-02-04T22:11:53Z</dc:date>
+        <rdfs:label xml:lang="en">meat (tenderized)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004169 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004169">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001180"/>
+        <obo:IAO_0000115 xml:lang="en">The chicken wings which have been prepared.</obo:IAO_0000115>
```

Comment:
Drop "The".  
I think its better to label this "prepared chicken wing".  Then "party wing" can be a narrow synonym.  As well you can place existing "Buffalo wing" under it. Note chicken wing is defined in singular.

Reply:
Changed the label. And also made 'chicken wing' a parent term in addition to 'prepared food product'.
'chicken wing' & 'prepared food product' (parent terms)
---prepared chicken wing
---------buffalo wing